### PR TITLE
partialLoop notice #7450

### PR DESCRIPTION
**Pull request by vixriihi**
Here is a test and a fix that will show the notice in the nested partialloop #7450

### CODE REVIEW

File: tests/ZendTest/View/Helper/PartialLoopTest.php [hunk 1]
```diff
@@ -348,6 +348,36 @@ public function testNestedCallsShouldNotOverrideObjectKey()
...
+            );
+
+        $view = new View();
+        $view->resolver()->addPath($this->basePath . '/application/views/scripts');
+        $this->helper->setView($view);
+
+        $result = $this->helper->__invoke('partialLoopParentObject.phtml', $data);
+        $this->assertContains('foo1', $result, $result);
+        $this->assertContains('foo2', $result, $result);
+
```
Comment (samsonasik) [inline tests/ZendTest/View/Helper/PartialLoopTest.php]: remove unused blank line before close brace <code>}</code>

File: tests/ZendTest/View/Helper/PartialLoopTest.php [hunk 2]
```diff
@@ -348,6 +348,36 @@ public function testNestedCallsShouldNotOverrideObjectKey()
...
+                    'message' => 'foo1',
+                    'data' =>
+                        array(
+                            array(
+                                'message' => 'foo2'
+                            )
+                        ),
+                    )
+                )
+            );
```
Comment (malukenho) [inline tests/ZendTest/View/Helper/PartialLoopTest.php]: align?

**Comment by adamlundrigan** [comment]
This looks good to me, with just one question:  The current behaviour is a notice is emitted and `objectKey` is set to null.  In your code the existing value of `objectKey` will remain unchanged.  I'm not too familiar with `partialLoop` so I'm unsure of whether or not this presents an issue.  If `objectKey` is always NULL preceding this point in it's execution then it does not, but if `objectKey` is reused through the nested stack it could be an issue.

**Comment by vixriihi** [comment]
Nether am I :) But the existing ```objectKey``` is set to null just before checking if the key exist [on this line](https://github.com/vixriihi/zf2/blob/zf7450/library/Zend/View/Helper/PartialLoop.php#L129), so it shouldn't be a problem.

**Pull request merged**